packit_service/worker/helpers/sidetag.py

```diff
@@ -109,6 +110,8 @@ def get_sidetag(cls, sidetag_group: str, dist_git_branch: str) -> Sidetag:
             PackitException if the sidetag doesn't exist either in the database or in Koji.
         """
         group = SidetagGroupModel.get_or_create(sidetag_group)
+        # resolve branch aliases, e.g. rawhide -> main
+        [dist_git_branch] = get_branches(dist_git_branch)
         if not (sidetag := group.get_sidetag_by_target(dist_git_branch)):
```

Comment:
```suggestion
        [dist_git_branch] = get_branches(dist_git_branch)
        if not (sidetag := group.get_sidetag_by_target(dist_git_branch)):
```

Not sure if I follow. This works for `rawhide` that maps only on `main`. Could someone use one of the other aliases that map on more branches? If yes then shouldn't be a loop?

Reply:
You are confusing branch aliases in dist-git and branch aliases in Packit. The loop that iterates over expanded Packit aliases is e.g. here:

https://github.com/packit/packit-service/blob/99e544d7fc564457e8e9efe2275dfc4f62612a2e/packit_service/worker/handlers/distgit.py#L968-L976

Reply:
I tried to stop the automerge but failed :sweat_smile: Hopefully we are on the same page now.

Reply:
> The loop that iterates over expanded Packit aliases is e.g. here:

Actually, in this case there are no aliases, it's a dist-git branch from the trigger event. Aliases are expanded and iterated over e.g. here:
https://github.com/packit/packit-service/blob/725cd285ed6d64946e16b5dad9918447f6808ebd/packit_service/worker/handlers/distgit.py#L1325-L1328